### PR TITLE
(PORT) Closet/Crate Examining for Observers

### PR DESCRIPTION
**Pull request by xPokee**
## About The Pull Request
_**PORTED FROM FALLOUT 13 - I COULD NOT FIND THE EXACT PR IT WAS ADDED IN**_
Lets ghosts click on lockers and crates to see what is inside of them.

## Why It's Good For The Game
It helps admins out in two ways, one, they don't have to VV to see what is inside of things, and two, they won't be asked by players to see what is inside of crates or lockers.

## Proof Of Testing
It didn't runtime or have any errors when I locally tested it, screenshots of it in action can be found below.

<details>
<summary>Screenshots/Videos</summary>
image
image

</details>

## Changelog

:cl:
qol: made it so you can examine what is inside of crates and lockers as an observer
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -355,6 +355,12 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 			. += span_notice("Swipe your PDA with an ID card/Just ID to change access levels.")
 			. += span_notice("Use multitool to [access_locked ? "unlock" : "lock"] the access panel.")
 
+//BUBBERSTATION EDIT START - LOCKER CONTENT EXAMINING
+	if(isobserver(user))
+		. += span_info("It contains: [english_list(contents)].")
+		investigate_log("had its contents examined by [user] as a ghost.", INVESTIGATE_GHOST)
+//BUBBERSTATION EDIT END - LOCKER CONTENT EXAMINING
```
Comment (Arturlang) [inline code/game/objects/structures/crates_lockers/closets.dm:362]: Can you override the examine proc on lockers in a zubbers folder, for example in modular_zubbers/code/objects/structures/crate_lockers/closets.dm , and overide the examine() proc there, and if you call parent at the start of that, it makes it nice and modular

**Review by Arturlang — CHANGES_REQUESTED** [review]

**Review by Arturlang — APPROVED** [review]
That was quick, nice work!

**Comment by nikothedude** [comment]
cant this be put on tg

File: modular_zubbers/code/game/objects/structures/crates_lockers/closets.dm [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/obj/structure/closet/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		. += span_info("It contains: [english_list(contents)].")
+		investigate_log("had its contents examined by [user] as a ghost.", INVESTIGATE_GHOST)
```
Comment (lessthnthree) [inline modular_zubbers/code/game/objects/structures/crates_lockers/closets.dm:5]: Do we really want an investigate log generated every time someone examines a locker?

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
Remove the logging every time a ghost examines a closet. It's not really an issue on our server and it just seems like a lot of logging bloat for logs I doubt will ever be used.

File: modular_zubbers/code/game/objects/structures/crates_lockers/closets.dm (hunk 2 above)
Comment (xPokee) [reply modular_zubbers/code/game/objects/structures/crates_lockers/closets.dm]: Alright, it's removed.

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**